addons/pos_pinelabs/models/pinelabs_pos_request.py

```diff
@@ -0,0 +1,74 @@
+import json
+import logging
+import requests
+
+from odoo import _
+
+REQUEST_TIMEOUT = 10
+PINELABS_AUTO_CANCEL_DURATION_MIN = 10
+
+_logger = logging.getLogger(__name__)
+
+class PinelabsPosRequest:
+
+    def __init__(self, payment_method):
+        self.pm = payment_method
+
+    def _get_pinelabs_url(self):
+        if self.pm.pinelabs_test_mode:
+            if self.pm.env['ir.config_parameter'].sudo().get_param('pos_pinelabs.pinelabs_proxy_endpoint', ''):
+                return self.pm.env['ir.config_parameter'].sudo().get_param('pos_pinelabs.pinelabs_proxy_endpoint')
+            _logger.warning('To test Pine Labs, the PineLabs Proxy Endpoint must be set because the Pine Labs test URL is only accessible in India and Malesia.')
+            return 'https://www.plutuscloudserviceuat.in:8201/API/CloudBasedIntegration/V1/'
+        return 'https://www.plutuscloudserviceuat.in:8201/API/CloudBasedIntegration/V1/' #TODO: We will update this with the production endpoint once it becomes available.
```

Comment:
```suggestion
        return 'https://www.plutuscloudserviceuat.in:8201/API/CloudBasedIntegration/V1/' #TODO: We will update this with the production endpoint once it becomes available.
```

We'll likely forget this TODO. Can't you put the url now?

Reply:
Hello @caburj 

We don't have the production URL at the moment. We will receive the production URL when we are ready to go live. Therefore, we need to wait until we go live to obtain the production URL. I assure you that I will update the production URL as soon as we receive it.